vmdb/app/helpers/provider_foreman_helper/textual_summary.rb

```diff
@@ -0,0 +1,37 @@
+module ProviderForemanHelper
+  module TextualSummary
```

Comment:
If we're only using this module in one place, then we should put the methods inside `ProviderForemanHelper`